harness/determined/launch/deepspeed.py

```diff
@@ -7,20 +7,74 @@
 import logging
```

Comment:
I think everything in this file is good, already (so we can revert this)

Reply:
I did figure that out earlier and attempted to revert, but apparently not :-)